src/routes/oath.ts

```diff
@@ -162,4 +164,65 @@ router.get("/hubspot_refresh_token", async (req, res) => {
 	}
 });
 
+
+export const ApiCall:any = async function (myapifunc:Function,refresh_token:string)
```

Comment:
This function only handles Hubspot access tokens, so maybe a new name to describe that?